Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/ConstantViewSize.cs

```diff
@@ -69,48 +136,15 @@ public float FovScale
             }
         }
 
+        #endregion
+
+        private const float ScalePower = 1f;
```

Comment:
Also noticed this, but I think that this is only used in a single location and it look like:

MixedRealityToolkit.SDK/Features/Utilities/Solvers/ConstantViewSize.cs:                float scale = Mathf.Clamp(fovScalar * Mathf.Pow(distance, ScalePower), minScale, maxScale);

Which means, AFAICT, that it just takes distances, does a distance^1, which basically ends up doing nothing.

Since we're already in here, it might be worth cleaning that up (deleting it)

Reply:
Good catch! I love deleting code like this